github/pull_request.rb

```diff
@@ -105,6 +113,15 @@ def check_commits_style
     end
   end
 
+  def add_issue_links
+    if new? && issue_numbers.any?
+      message = issue_numbers.inject("\nIssues:") do |msg, issue_number|
+        msg + " [##{issue_number}](http://projects.theforeman.org/issues/#{issue_number})"
+      end
+      append_pull_request_body(message)
```

Comment:
This absolutely does not modify the commit message, only the pull request description on a new PR with issue numbers in the commit message.